### PR TITLE
Update Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs

### PR DESCRIPTION
**Pull request by GrzegorzDryja**
Update made thanks to PM suggestions.
1. I actually thought that this function could be used for the entire project
2. I think I did it
3. At the moment, no matter, it probably disappeared
4. Ok thanks
5. No problem
6. I meant the future GUI interface ;)

### CODE REVIEW

File: Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs [hunk 1]
```diff
@@ -6,25 +6,23 @@ namespace Notatnik_parser.Notepad
 {
     public class NotepadMenu
     {   
-        string note;
-
-        public void SaveFile()
+        private string SaveFile()
```
Comment (tomku-mar) [inline Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs]: now, this methos is responsible for two functionalities: get text from console and save it to file. Try create private method for get text and add string parameter to SaveFile function. Then you can use them in Run()

File: Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs [hunk 2]
```diff
@@ -6,25 +6,23 @@ namespace Notatnik_parser.Notepad
...
 
         public void Run()
         {
             Console.WriteLine("Witamy w notatniku. Wprowadź notatkę (wciśnij <enter> aby zapisać):");
-            note = Console.ReadLine();
-
-            ConsoleKeyInfo saveButton = Console.ReadKey();
-            if (saveButton.Key == ConsoleKey.Enter)
+            
+            ConsoleKeyInfo forSave = Console.ReadKey();
```
Comment (tomku-mar) [inline Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs]: I think the name is still not clear. What "forSave" tells you? What is forSave here? Name should tell you exacly function of the varible. Maybe somthing like "typedKey"? Try find different name ;)

**Review by tomku-mar — CHANGES_REQUESTED** [review]
Naming variables and functions is very important and hard. I have problems with it myself

**Comment by GrzegorzDryja** [comment]
Ok, now I get it. "typedKey" name for this variable is actually very good
name in this case.

śr., 7 lis 2018 o 17:56 tomkowy <notifications@github.com> napisał(a):

**Comment by GrzegorzDryja** [comment]
I know, I've watch few lessons about this and one speaker on Code
Europe emphasized
it strongly. But practice is different. Thank you for looking on code from
your perspective.

śr., 7 lis 2018 o 18:00 tomkowy <notifications@github.com> napisał(a):

File: Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs (hunk 2 above)
Comment (GrzegorzDryja) [reply Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs]: Oh, mail does not goes where I supposed.

**Comment by GrzegorzDryja** [comment]
I've made some changes, from that moment please check "notepad" branch for merge.

File: Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs [hunk 3]
```diff
@@ -5,27 +5,35 @@
...
-            Console.WriteLine("Witamy w notatniku. Wprowadź notatkę (wciśnij <enter> aby zapisać):");
-            
-            ConsoleKeyInfo forSave = Console.ReadKey();
-            if (forSave.Key == ConsoleKey.Enter)
+        {   
+            string note;
+
+            Console.WriteLine("Witamy w notatniku. Wprowadź notatkę (wciśnij 2 x <enter> aby zapisać):");
+
+            note = Console.ReadLine();
```
Comment (tomku-mar) [inline Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs:26]: You can assign a variable in one line:
string note = Console.ReadLine();
Youre code is fine, but i think that my proposition is more readable (one line of code less)

**Review by tomku-mar — APPROVED** [review]
:)

File: Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs (hunk 3 above)
Comment (GrzegorzDryja) [reply Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs:26]: Yes, I know. BTW Most of all I like to use this kind of text format - while curly braces is on the end of the line - it shorten number of lines a lot, [check this commit](https://github.com/tomkowy/notatnik_parser/blob/issue%2322-curly-braces-on-the-end/Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs)
To work with this project I switched to match other files :)

**Comment by GrzegorzDryja** [comment]
Finally merging my branch made me very proud of myself!

**Pull request merged**